src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -543,6 +544,12 @@ public function get_test_theme_version() {
 		}
 
 		foreach ( $all_themes as $theme_slug => $theme ) {
+
+			if ( $theme->get( 'Template' ) && $theme->get( 'Template' ) !== $theme_slug ) {
```

Comment:
Wouldn't the `::get_template()` method be better to use?

```suggestion
			if ( $theme->get_template() && $theme->get_template() !== $theme_slug ) {
```

Reply:
yes :-)